docs/team/seantehds.md

```diff
@@ -8,40 +8,41 @@ title: Teh De Sheng Sean's Project Portfolio Page
 TaskWise is a project management application used by CS2103/T project managers.
 
 The user interacts with it via the Command Line Interface (CLI) and it has a Graphical User
-Interface (GUI) made using JavaFX. It is written in Java, and it has about 13848 LoC.
+Interface (GUI) made using JavaFX. It is written in Java, and it has about 14107 LoC.
 
 Given below are my contributions to the project.
 
 ### New Features and Components
 
-* **New Feature**: Mark Command: [#61](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/61)
+* **New Feature:** Mark Command: [#61](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/61)
     * What it does: Marks a task within the task list as completed.
     * Justifications: Allows the user to mark tasks as completed to allow the task manager to keep track of
       tasks and whether they are finished or not.
     * Highlights: The task reduces in opacity when it is marked as completed.
     * Credits: Sebastian for implementing the Status class and field necessary for tracking whether the
```

Comment:
Could you help me change this line over to this particular line:
Credits: Sebastian for implementing the Status class and field necessary for tracking whether the task has been completed or not, as well as the change in opacity of the task card depending on status.